shell_automaton/src/action.rs

```diff
@@ -232,6 +236,11 @@ pub enum Action {
     ProtocolRunnerSpawnServerError(ProtocolRunnerSpawnServerErrorAction),
     ProtocolRunnerSpawnServerSuccess(ProtocolRunnerSpawnServerSuccessAction),
 
+    ProtocolRunnerCurrentHeadInit(ProtocolRunnerCurrentHeadInitAction),
```

Comment:
@sebastiencs I would rename these so that it is not ambiguous (LatestCommitHashes or LatestContextHashes, so it is not confused with the "current head" concept from the shell).